.travis.yml

```diff
@@ -95,3 +95,4 @@ cache:
     - ~/.npm
     - ~/.cache
     - ~/.travis_cache/
+    - superset/assets/node_modules/.cache/
```

Comment:
Should we scope to `superset/assets/node_modules/.cache/terser-webpack-plugin`?

Reply:
Could do, though there's nothing else in there at the moment. I'm not sure what else may use this if anything.

Reply:
not sure but maybe keep it more specific to avoid the cache expanding because of something else that we are not using.